send_review_notification/github_services.py

```diff
@@ -0,0 +1,266 @@
+CREATE_DISCUSSION_URL_TEMPLATE = (
+    'https://api.github.com/orgs/{0}/teams/{1}/discussions')
+
+
+def init_service(token: Optional[str]=None) -> None:
+    """Initialize service with the given token."""
+    if token is None:
+        raise Exception('Must provide Github Personal Access Token.')
+
+    global _TOKEN # pylint: disable=global-statement
+    _TOKEN = token
+
+
+# Here we use type Any because.
+def check_token(func: Callable[..., Any]) -> Callable[..., Any]:
+    """A decorator to check whether the service is initialized with
+    the token.
+    """
+
+    # Here we use type Any because.
+    def execute_if_token_initialized(*args: Any, **kwargs: Any) -> Any:
+        """Executes the given function if the token is initialized."""
+        if _TOKEN is None:
+            raise Exception(
+                'Initialize the service with '
+                'github_services.init_service(TOKEN).')
+        return func(*args, **kwargs)
+
+    return execute_if_token_initialized
+
+
+def _get_request_headers() -> Dict[str, str]:
+    """Returns the request headers for github-request."""
+    return {
+        'Accept': 'application/vnd.github.v3+json',
+        'Authorization': f'token {_TOKEN}'
+    }
+
+
+@check_token
+def get_prs_assigned_to_reviewers(
+    org_name: str,
+    repository: str,
+    wait_hours: int
+) -> DefaultDict[str, List[github_domain.PullRequest]]:
+    """Fetches all the PRs on the given repository and returns a list of PRs
+    assigned to reviewers.
+    """
+
+    pr_url = PULL_REQUESTS_URL_TEMPLATE.format(org_name, repository)
+    reviewer_to_assigned_prs: (
+        DefaultDict[str, List[github_domain.PullRequest]]) = (
+        collections.defaultdict(list))
+
+    page_number = 1
+    while True:
+        logging.info('Fetching Pull requests')
+        params: Dict[str, Union[str, int]] = {
+            'page': page_number, 'per_page': 100, 'status': 'open'
+        }
+        response = requests.get(
+            pr_url,
+            params=params,
+            headers=_get_request_headers()
+        )
+        response.raise_for_status()
+        pr_subset = response.json()
+
+        if len(pr_subset) == 0:
+            break
+        page_number += 1
+
+        pull_requests = [
+            github_domain.PullRequest.from_github_response(pull_request)
+            for pull_request in pr_subset]
+        update_assignee_timestamp(org_name, repository, pull_requests)
+        for pull_request in pull_requests:
+            if not pull_request.is_reviewer_assigned():
+                continue
+            for reviewer in pull_request.assignees:
+                pending_review_time = (
+                    datetime.datetime.now(datetime.timezone.utc) -
+                    reviewer.timestamp)
+                if (reviewer.name != pull_request.author) and (
+                        pending_review_time >=
+                        datetime.timedelta(hours=wait_hours)
+                    ):
```

Comment:
```suggestion
                    pending_review_time >=
                    datetime.timedelta(hours=wait_hours)
                ):
```

this doesn't need to be indented so much

Reply:
done